governance/contributor-ladder.md

```diff
@@ -0,0 +1,111 @@
+Within the [`roles.md`](roles.md) file, you will find various volunteer positions available.
+To begin, consider taking ownership of a small initiative within the TAG, such as opening an issue that was discussed during a meeting, or taking on responsibilities like reviewing pull requests (PRs).
+Additionally, you can contribute by raising awareness about the TAG ENV among your network. This can involve sharing information and updates, presenting talks at conferences or events, organizing meetups, or even writing a blog post (refer to [blog post process](blogpost-process.md) for guidance).
+Collaboration with other organizations is also a priority for us, so actively seeking out open source projects or organizations you know in the cloud native sustainability space and connecting them with the TAG would be highly valuable.
+By proactively driving topics that directly benefit the Environmental Sustainability TAG, you significantly contribute to its growth and overall impact and will help you mature as a contributor.
+
+* Proactively start discussions and take ownership of smaller initiatives
+* Establish yourself in the community
+* Get an overview of processes
+
+## Leading a project
+
+Once you have established connections within the TAG and actively participated in discussions and projects, you may encounter a topic that captures your interest and warrants further attention.
+This particular project might require more time and dedication from your side as well as the support of other individuals within the community.
+If you have a project idea that aligns with the objectives of the TAG, consider writing a proposal, in a GitHub issue, outlining its motivation, scope, goals, and deliverables.
+Present this proposal during a TAG meeting to initiate a discussion and gather feedback and support from fellow members.
+Taking on a [project leadership role](roles.md#role-of-project-leads) within the community project becomes your next significant step towards leadership recognition within the group.
+
+* Take ownership of a project within the TAG
+* Lead a community group within the TAG
+
+## Helping out
+
+In open source, newcomers join the community seeking guidance and support as they embark on their journey, just as you once did.
+While it is essential to provide support to current members and leaders, it is equally important to focus on assisting and orienting newcomers.
+Modeling empathy and offering support to the community as a whole should be at the forefront of your actions.
+It is crucial to embody the values of the community as outlined in the [code of conduct](http://cncf.io/conduct/).
+
+There are various ways you can contribute to supporting and empowering new contributors.
+Firstly, you can actively participate by stepping in and offering mentorship and guidance when someone requires assistance. This could involve helping them navigate the community, providing technical guidance, or answering their questions.
```

Comment:
```suggestion
You can actively participate by offering mentorship and guidance when someone requests assistance. This could involve helping them navigate the community, providing technical guidance, or answering their questions.
```